eng/scripts/Save-Package-Namespaces-Property.ps1

```diff
@@ -0,0 +1,103 @@
+<#
+.SYNOPSIS
+Given an artifact staging directory and the artifacts list, loop through all artifacts
+that docs that aren't skipping docs creation and add the namespace to the PackageInfo
+json file.
+
+.DESCRIPTION
+Given an artifact staging directory and the artifacts list, loop through all artifacts
+that docs that aren't skipping docs creation and add the namespace to the PackageInfo
+json file. This is done using the .whl file from the build.
+
+.PARAMETER ArtifactStagingDirectory
+The root directory of the staged artifacts. The PackageInfo files will be in the
+PackageInfo subdirectory. The whl files are going to be in the subdirectory which
+is the same as the artifact's name but artifact name in the file's name will have
+ underscores instead of dashes.
+
+.PARAMETER ArtifactsList
+The list of artifacts to gather namespaces for, this is only done for libraries that are
+producing docs.
+-ArtifactsList '${{ convertToJson(parameters.Artifacts) }}' | ConvertFrom-Json
```

Comment:
```suggestion
-ArtifactsList ('${{ convertToJson(parameters.Artifacts) }}' | ConvertFrom-Json)
```

Not that this matters, but given you mentioned the application of this snippet was hard to debug, might as well fix the comment :D

Reply:
@benbp, I pushed another commit and made this update to the comment. It wasn't hard to debug, it was just easy to forget the parens in the yml...